test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 coverage
 nose
+requests
```

Comment:
Maybe also good to have an `extras_require` section in `setup.py` so that one can depend on e.g. `katdal[s3]` or `katdal[rados]` in other packages.

Reply:
Yes, I was wondering about that mechanism and waiting for someone to explain how it works :-)

Reply:
Done.